app/code/Magento/Inventory/Setup/InstallSchema.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+namespace Magento\Inventory\Setup;
+
+use Magento\Framework\DB\Ddl\Table;
+use Magento\Framework\Setup\InstallSchemaInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\SchemaSetupInterface;
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\InventoryApi\Api\Data\SourceCarrierLinkInterface;
+
+class InstallSchema implements InstallSchemaInterface
+{
+
+    /**
+     * Constant for table names of the model \Magento\Inventory\Model\Source
+     */
+    const TABLE_NAME_SOURCE = 'inventory_source';
+
+    /**
+     * Constant for table name of \Magento\Inventory\Model\SourceCarrierLink
+     */
+    const TABLE_NAME_SOURCE_CARRIER_LINK = 'inventory_source_carrier_link';
+
+    /**
+     * Constant for decimal precision for latitude and longitude
+     */
+    const LATLON_PRECISION = 10;
+    const LATLON_SCALE = 6;
```

Comment:
Use DECIMAL(8,6) for latitude (90 to -90 degrees) and DECIMAL(9,6) for longitude (180 to -180 degrees). 6 decimal places is fine for most applications. Both should be "signed" to allow for negative values.